### PR TITLE
Update .gitignore

### PR DESCRIPTION
**Pull request by AliAlbabily**
v1.2

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
+# v1.2
 # Created by https://www.toptal.com/developers/gitignore/api/java,windows,intellij
 # Edit at https://www.toptal.com/developers/gitignore?templates=java,windows,intellij
```
Comment (hanisss) [inline .gitignore:4]: test

**Review by hanisss — COMMENTED** [review]
test

**Comment by GlamnikiA** [comment]
Nice

File: .gitignore (hunk 1 above)
Comment (DeGGiii) [reply .gitignore:4]: Hej

**Comment by Madzic8** [comment]
test

**Pull request merged**